### PR TITLE
fix description message of command line options

### PR DESCRIPTION
**Pull request by huang195**
Several command line options' descriptions were obviously copied and pasted, resulting in inconsistent description of these options.

### CODE REVIEW

File: pilot/cmd/pilot-discovery/main.go [hunk 1]
```diff
@@ -114,9 +114,9 @@ func init() {
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPMaxMessageSize, "mcpMaxMsgSize", bootstrap.DefaultMCPMaxMsgSize,
 		"Max message size received by MCP's grpc client")
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPInitialWindowSize, "mcpInitialWindowSize", bootstrap.DefaultMCPInitialWindowSize,
-		"Max message size received by MCP's grpc client")
+		"Initial window size for the gRPC connection")
 	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPInitialConnWindowSize, "mcpInitialConnWindowSize", bootstrap.DefaultMCPInitialConnWindowSize,
-		"Max message size received by MCP's grpc client")
+		"Initial connection window size for the gRPC connection")
```
Comment (howardjohn) [inline pilot/cmd/pilot-discovery/main.go]: nit: `MCP's gRPC connection` may be more clear, pilot will have grpc connections for XDS as well

**Comment by howardjohn** [comment]
feel free to remove the hold if you don't want to address the nitpick, just didn't want it automerged

File: pilot/cmd/pilot-discovery/main.go (hunk 1 above)
Comment (huang195) [reply pilot/cmd/pilot-discovery/main.go]: Fixed. Definitely make it more clear.

**Pull request merged**